pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -148,6 +150,16 @@ func updateWorkspaceStorageConfig(devEnvironments *chev2.CheClusterDevEnvironmen
 	return nil
 }
 
+func updatePersistUserHomeConfig(persistentHomeConfig *chev2.PersistentHomeConfig, workspaceConfig *controllerv1alpha1.WorkspaceConfig) {
+	workspaceConfig.PersistUserHome = nil
+	if persistentHomeConfig != nil {
+		if workspaceConfig.PersistUserHome == nil {
+			workspaceConfig.PersistUserHome = &controllerv1alpha1.PersistentHomeConfig{}
+		}
```

Comment:
This conditional is a no-op, since you set `workspaceConfig.PersistUserHome = nil` above.

Reply:
Good catch :sweat_smile: